File: modules/angular2/test/core/pipes/spies.dart
library pipes.spies;

import 'package:angular2/src/core/change_detection/change_detection.dart';
import 'package:angular2/test_lib.dart';

@proxy
class SpyChangeDetectorRef extends SpyObject implements ChangeDetectorRef {
  noSuchMethod(m) => super.noSuchMethod(m);
}

File: modules/angular2/test/core/pipes/spies.ts
import {ChangeDetectorRef_} from 'angular2/src/core/change_detection/change_detector_ref';

import {SpyObject, proxy} from 'angular2/testing_internal';

export class SpyChangeDetectorRef extends SpyObject {
  constructor() { super(ChangeDetectorRef_); }
}